.github/workflows/pr_tests.yml

```
@@ -18,86 +18,88 @@ env:
 jobs:
   run_tests_cpu:
     name: CPU tests on Ubuntu
-    runs-on: [ self-hosted, docker-gpu ]
+    runs-on: [self-hosted, docker-gpu]
     container:
       image: python:3.7
       options: --shm-size "16gb" --ipc host -v /mnt/hf_cache:/mnt/cache/
 
     steps:
-    - name: Checkout diffusers
-      uses: actions/checkout@v3
-      with:
-        fetch-depth: 2
-
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install torch --extra-index-url https://download.pytorch.org/whl/cpu
-        python -m pip install -e .[quality,test]
-
-    - name: Environment
-      run: |
-        python utils/print_env.py
-
-    - name: Run all fast tests on CPU
-      run: |
-        python -m pytest -n 2 --max-worker-restart=0 --dist=loadfile -s -v --make-reports=tests_torch_cpu tests/
-
-    - name: Failure short reports
-      if: ${{ failure() }}
-      run: cat reports/tests_torch_cpu_failures_short.txt
-
-    - name: Test suite reports artifacts
-      if: ${{ always() }}
-      uses: actions/upload-artifact@v2
-      with:
-        name: pr_torch_cpu_test_reports
-        path: reports
+      - name: Checkout diffusers
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install torch --extra-index-url https://download.pytorch.org/whl/cpu
+          python -m pip install -e .[quality,test]
```

```suggestion
          python -m pip install git+https://github.com/huggingface/accelerate
          python -m pip install -e .[quality,test]
```
This should work I think :-)

---

I tried putting the accelerate from master after the `quality,test` to ensure this is the version we will keep for the tests.